src/main/java/com/yugabyte/sample/apps/AppConfig.java

```diff
@@ -125,8 +127,8 @@
   // Name of the default username for postgres.
   public String defaultPostgresUsername = "postgres";
 
-  // Does the table need to be dropped.
-  public boolean shouldDropTable = false;
+  // should we drop / truncate table
+  public AppBase.TableOp tableOp = TableOp.TruncateTable;
```

Comment:
@tedyu -- my understanding was that we wanted the truncate behavior only for YSQL (because with SQL, simply rerunning the tests will result in duplicate key exceptions). With YCQL, it has UPSERT semantics by default -- so let's us not change the default behavior for YCQL. We could have the --truncate option, but perhaps not make it the default.